R/plot_grid.R

```diff
@@ -177,8 +177,10 @@ plot_grid <- function(..., plotlist = NULL, align = c("none", "h", "v", "hv"),
   if (is.null(cols)) cols <- ceiling(num_plots/rows)
   if (is.null(rows)) rows <- ceiling(num_plots/cols)
 
-  # if the user wants to layout the plots by column, we use the calculated rows to reorder plots
-  if (!isTRUE(byrow)) plots <- plots[c(t(matrix(c(1:num_plots, rep(NA, (rows * cols) - num_plots)), nrow = rows, byrow = FALSE)))]
+  # if the user wants to layout the plots by column, we use the calculated rows to reorder plots and labels
+  if (!isTRUE(byrow)) 
+    plots <- plots[c(t(matrix(c(1:num_plots, rep(NA, (rows * cols) - num_plots)), nrow = rows, byrow = FALSE)))]
+    labels <- labels[c(t(matrix(c(1:num_plots, rep(NA, (rows * cols) - num_plots)), nrow = rows, byrow = FALSE)))]
```

Comment:
Shouldn't this code block be enclosed in curly braces? Otherwise line 183 gets executed regardless of whether the condition is true or not, no? Either way, please add curly braces for clarity.